laborator/content/toolchain-decompilare/demo/compiling_steps/hello-world.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
```

Comment:
Are those credits necessary for a IOCLA demo?

Reply:
These are not general demos, they are more particular to me. I put this signature so I can identify them more easily.